src/pages/resultsView/querySummary/QuerySummaryUtils.tsx

```diff
@@ -64,18 +64,18 @@ export function getAlterationSummary(
     }
 }
 
-export function getStudyViewFilterHash(
+export function submitToStudyViewPage(
+    queriedStudies: Pick<CancerStudy, "studyId">[],
     samples: Pick<Sample, "sampleId" | "studyId">[],
     hasVirtualStudies: boolean,
     sampleLists?: Pick<SampleList, "category">[]) {
     const hasAllCaseLists = _.some(sampleLists, sampleList => sampleList.category === 'all_cases_in_study');
-    let studyViewFilterHash: string | undefined;
     if (samples.length > 0 && (hasVirtualStudies || !hasAllCaseLists)) {
         const sampleIdentifiers = samples.map(sample => ({
             sampleId: sample.sampleId,
             studyId: sample.studyId
         }));
-        studyViewFilterHash = `filterJson=${JSON.stringify({ sampleIdentifiers })}`;
+        window.localStorage.setItem("legacyStudyViewPageFilter", `filterJson=${JSON.stringify({ sampleIdentifiers })}`);
```

Comment:
so this is where i think we use the the data passing trick we talked about in slack instead of localStorage.